src/components/CreateCategory.vue

```diff
@@ -0,0 +1,65 @@
+<template >
+    <v-btn @click="See" color="primary"
```

Comment:
Please be consistent and start with small case. You can also simplify and remove functions see() and close by setting the boolean in the @click event handler
```suggestion
    <v-btn @click="dialog = true" color="primary"
```